index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500;700&family=Open+Sans&display=swap"
+        rel="stylesheet">
+    <title>Resume_project1</title>
+</head>
+
+<body>
+    <div class="namewrapper">
+        <h1 class="name">Adam Dworetzky</h1>
+        <div class="contactwrapper">
+            <p class="contact">561-305-5638</p>
+            <p class="contact">adamdworetzky.com</p>
+            <p class="contact">Adam.Dworetzky@gmail.com</p>
```

Comment:
For an HTML resume, it makes sense for these things to be links; for your email, you do that by using [`mailto`](https://css-tricks.com/snippets/html/mailto-links/):
```suggestion
            <p class="contact"><a href="adamdworetzky.com">adamdworetzky.com</a></p>
            <p class="contact"><a href="mailto:Adam.Dworetzky@gmail.com">Adam.Dworetzky@gmail.com</a></p>
```